components/RichText/Quote.module.scss

```diff
@@ -33,10 +33,8 @@
 }
 
 .content {
-    @include heading-2;
-
+    font-style: italic;
     flex: 1;
-    color: $color-base-700;
     text-align: center;
```

Comment:
Please test with the editor to see that the text in the quote indeed follows the text alignment settings.

Reply:
texted all alignment options and it follows the proposed design inside figma